com.github.Inochi2D.Inochi-Creator.yml

```diff
@@ -0,0 +1,164 @@
+id: com.github.Inochi2D.Inochi-Creator
+runtime: org.kde.Platform
+runtime-version: "5.15-21.08"
+sdk: org.kde.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.ldc
+  - org.freedesktop.Sdk.Extension.dmd
+command: "inochi-creator"
+finish-args:
+    - "--device=all" #OpenGL rendering, webcams (may be required? not sure)
+    - "--socket=x11"
+    - "--share=ipc"
+    - "--socket=wayland"
+    - "--filesystem=host" #This is a dev tool, it should probably have as much access to the host as it can.
+    - "--filesystem=home" #So that it can access the project files, should they be located in the home folder
+    - "--filesystem=~/inochi-creator-crashdump.txt:rw" # Error logs

```

Comment:
this doesn't work. It can be a file. Also since you already grant `home`... it pointless.

Reply:
I have the permission set, specifically, just in case the user revokes home/host access when portal use is implemented. Easier to set it specifically and let the user come to the conclusion that it's more or less required for logs, so that they don't remove the permission.


Also, why would it not work? Would I need to use :create instead?

Reply:
I meant "It can NOT be a file". Again, no point having it.